src/platform/graphics/texture.js

```diff
@@ -25,6 +25,26 @@ let id = 0;
  * A texture is a container for texel data that can be utilized in a fragment shader. Typically,
  * the texel data represents an image that is mapped over geometry.
  *
+ * Note on **HDR texture format** support:
+ * 1. **As textures**:
+ * - float (e.x. {@link PIXELFORMAT_RGBA32F}), half-float (e.x. {@link PIXELFORMAT_RGBA16F}) and
+ * small-float ({@link PIXELFORMAT_111110F}) formats are always supported on both WebGL2 and WebGPU
+ * with point sampling.
+ * - half-float and small-float formats are always supported on WebGL2 and WebGPU with linear sampling.
+ * - float formats are supported on WebGL2 and WebGPU with linear sampling only if
+ * {@link GraphicsDevice#textureFloatFilterable} is true.
```

Comment:
Shouldn't these bullets be indented? I guess 2 spaces is fine but check the output. Same for the bullets after 2.

Reply:
done, didn't even know this was possible ;)
thanks